assets/src/js/godam-player/managers/layers/hotspotLayerManager.js

```diff
@@ -52,7 +58,8 @@ export default class HotspotLayerManager {
 			}
 
 			const endTime = layerObj.displayTime + layerObj.duration;
-			const isActive = currentTime >= layerObj.displayTime && currentTime < endTime;
+			const isActive =
+		currentTime >= layerObj.displayTime && currentTime < endTime;
```

Comment:
Missed tab indentation.
```suggestion
			const isActive =
		        currentTime >= layerObj.displayTime && currentTime < endTime;
```